README.md

```diff
@@ -414,6 +414,19 @@ For more information about configuring Karma, check the [Karma documentation](ht
 
 ## Gotchas
 
+### Accessing dev server from outside `localhost`
+
+By default, Webpack disables access to the development server for hosts other than `localhost`, which means the development server will not be accessible from outside. If you want to give external access to the development server, you can set the `develop.disableHostCheck` to `true`:
```

Comment:
Maybe we can mention the `Invalid Host header` message. That is what people will probably search for.